docs/references/javascript/organization/invitations.mdx

```diff
@@ -49,7 +112,7 @@ function getPendingInvitations(params?: GetPendingInvitationsParams): Promise<Or
 | `limit` | `number` | Limit of the results returned. |
 | `offset` | `number` | The offset of results to start the retrieval. |
 
-### Returns
+### `Returns`
```

Comment:
```suggestion
### `getPendingInvitations` returns
```
Should be this, no?

Reply:
its completely removed in the beta docs so i didn't even want to touch this section, but i can update it for consistency in the core 1 docs, thats fine